server/simple_server.go

```diff
@@ -306,16 +304,15 @@ func (s *SimpleServer) Register(svcI Service) error {
 		// register all context endpoints with our wrapper
 		for path, epMethods := range mcs.JSONEndpoints() {
 			for method, ep := range epMethods {
-				endpointName := metricName(prefix, path, method)
+				fullPath := strings.TrimPrefix(prefix+path, "/")
```

Comment:
Maybe we still do the `prefix+path` when we call `TimedAndCounted`, but then do the `TrimPrefix` inside the `TimedAndCounted`?

(and yes, I agree the register repetition needs to go, but haven't thought of a good strategy yet. Ideas are welcome!)